substrate/frame/examples/basic/src/lib.rs

```diff
@@ -177,267 +188,199 @@ pub mod pallet {
 			Weight::zero()
 		}
 
-		// `on_finalize` is executed at the end of block after all extrinsic are dispatched.
+		// `on_finalize` is executed at the end of block after all extrinsics are dispatched.
 		fn on_finalize(_n: BlockNumberFor<T>) {
 			// Perform necessary data/state clean up here.
 		}
+	}
 
-		// A runtime code run after every block and have access to extended set of APIs.
-		//
-		// For instance you can generate extrinsics for the upcoming produced block.
-		fn offchain_worker(_n: BlockNumberFor<T>) {
-			// We don't do anything here.
-			// but we could dispatch extrinsic (transaction/unsigned/inherent) using
-			// sp_io::submit_extrinsic.
-			// To see example on offchain worker, please refer to example-offchain-worker pallet
-			// accompanied in this repository.
+	/// An example storage item to store a single value, in our case, some Balance.
+	/// This storage item uses [`OptionQuery`] by default which will return what is in the actual DB
+	/// state. If this is `None`, then the resulting query will be `None`.
```

Comment:
Sorry, but confusing wording. 

Simply, if value exists in state, it returns `Some(v)`, else `None`. repeating `None` does not help. 

Also, I would not name `DB` here but rather _state_, to be more generic. The runtime doesn't know or care if there is an actual database there, it only cares about the existence of some state provided by `sp_io::storage`.